### PR TITLE
Feature block

### PR DESCRIPTION
**Pull request by saurabhariyan**
- pending tests for routes

### CODE REVIEW

File: src/components/variables.css [hunk 1]
```diff
@@ -92,4 +92,12 @@
   --screen-sm-min: 768px;  /* Small screen / tablet */
   --screen-md-min: 992px;  /* Medium screen / desktop */
   --screen-lg-min: 1200px; /* Large screen / wide desktop */
+
+  /*
+   * Colors
+   * ======================================================================== */
+  --nyaaya_blue: rgb(232, 250, 255);
+  --nyaaya_green: rgb(84, 241, 178);
```
Comment (archanks) [inline src/components/variables.css]: Don’t describe colours in variable names. Refer to them as primary(green) and secondary(blue)

File: src/components/variables.css [hunk 2]
```diff
@@ -92,4 +92,12 @@
...
+  --background_grey: rgb(237, 237, 237);
```
Comment (archanks) [inline src/components/variables.css]: This would be tertiary

File: src/components/Tile/Tile.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+.tile {
+  padding: 0.5em;
+  background-color: var(--nyaaya_blue);
+  height: 200px;
+  width: 20%;
+  margin: 0.25em;
+  text-align: center;
+  text-decoration: none;
+  text-rendering: optimizeLegibility;
```
Comment (archanks) [inline src/components/Tile/Tile.css:11]: Why do this here? It would bea global setting

File: src/components/Tile/Tile.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@import '../variables.css';
+
+.tile {
+  padding: 0.5em;
+  background-color: var(--nyaaya_blue);
+  height: 200px;
```
Comment (archanks) [inline src/components/Tile/Tile.css]: Stick to one unit and use it consistently. Don’t use px and em

File: src/components/Tile/Tile.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import Link from '../Link';
+
+/* eslint-disable react/prop-types */
+class Tile extends React.Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    url: PropTypes.string.isRequired,
+  };
+  static defaultProps = {
+    name: null,
```
Comment (archanks) [inline src/components/Tile/Tile.js:15]: Why null?

File: src/components/Tile/Tile.test.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Tile from './Tile';
+
+describe('Tile component', () => {
+  it('should render tile with name and url', () => {
+  const component = mount(<Tile name="test" url="www.test.com" />, {context: { insertCss: () => {} }});
+  expect(component.text()).toEqual('test');
+  expect(component.find('Link').prop("to")).toEqual('www.test.com')
```
Comment (archanks) [inline src/components/Tile/Tile.test.js]: You can just maintain a snapshot and expect it to match that

File: src/components/Tile/Tile.test.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  });
+
+  it('should render tile with null values', () => {
```
Comment (archanks) [inline src/components/Tile/Tile.test.js]: Why? What is the point of an empty div with nothing in it?

File: src/routes/category/Category.css [hunk 8]
```diff
@@ -14,11 +14,22 @@
   font-weight: normal;
 }
 
-.categoryTopics {
-  margin: 0;
-  padding: 0;
+.featuredTopics {
```
Comment (archanks) [inline src/routes/category/Category.css]: Why is this featured?

File: src/routes/category/Category.js [hunk 9]
```diff
@@ -23,16 +23,10 @@ class Category extends React.Component {
...
-          <div className={s.container}>
-            {this.props.topics.map(topic => (
-              <div className={s.topic}>
-                <Link className={s.topicLink} to={topic.url}>
-                  {topic.name}
-                </Link>
-              </div>
-            ))}
-          </div>
+        <div className={s.featuredTopics}>
```
Comment (archanks) [inline src/routes/category/Category.js]: It’s just a topic

File: src/routes/home/Home.css [hunk 10]
```diff
@@ -75,7 +75,7 @@
   text-decoration: none;
   text-align: center;
 }
-
+*/
```
Comment (archanks) [inline src/routes/home/Home.css]: Remove commented unused code

File: package.json [hunk 11]
```diff
@@ -55,6 +55,8 @@
     "chokidar": "^1.7.0",
     "css-loader": "^0.28.7",
     "enzyme": "^3.2.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.1",
```
Comment (archanks) [inline package.json:59]: Why do you need all of this ?

File: jest.setup.js [hunk 12]
```diff
@@ -0,0 +1 @@
+import '@babel/polyfill';
```
Comment (archanks) [inline jest.setup.js:1]: Why?

File: enzyme.config.js [hunk 13]
```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```
Comment (archanks) [inline enzyme.config.js:4]: Why not use react-test-renderer that’s already there?

**Review by archanks — CHANGES_REQUESTED** [review]

File: package.json (hunk 11 above)
Comment (saurabhariyan) [reply package.json:59]: Adpater is needed for enzyme itself while enzyme to JSON gives more readable snapshots. I've used them on other snapshots tests not in this push.

File: src/routes/category/Category.css (hunk 8 above)
Comment (saurabhariyan) [reply src/routes/category/Category.css]: In the spirit of keeping the same class names in different CSS files. It is the same class also.

File: src/components/Tile/Tile.test.js (hunk 7 above)
Comment (saurabhariyan) [reply src/components/Tile/Tile.test.js]: It is testing that the default props are getting passed. Though these are required props so we should not have to use the default values ever

**Pull request closed without merging**